lib/tree-view.coffee

```diff
@@ -1056,6 +1056,9 @@ class TreeView
         # Drop event from OS
         for file in e.dataTransfer.files
           @moveEntry(file.path, newDirectoryPath)
+    else if e.dataTransfer.files.length
```

Comment:
Would `e.dataTransfer.files` be `undefined` ?

Reply:
Good question.  `files` is  standard DataTransfer property that will always be available. https://developer.mozilla.org/en-US/docs/Web/API/DataTransfer/files

Reply:
Thanks for the info. Any it says that `dataTransfer` exists on *all drag events*. Good News.